packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2243,7 +2249,7 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     properties.add(IterableProperty<String>('flags', flags, ifEmpty: null));
     properties.add(FlagProperty('isInvisible', value: isInvisible, ifTrue: 'invisible'));
     properties.add(FlagProperty('isHidden', value: hasFlag(SemanticsFlag.isHidden), ifTrue: 'HIDDEN'));
-    properties.add(StringProperty('label', _label, defaultValue: ''));
+    properties.add(StringProperty('label', _attributedLabel.string, defaultValue: ''));
```

Comment:
somehow we should probably also include the attributes in the debug description so they show up in e.g. the semantics tree.